config/locales/en.yml

```diff
@@ -3265,6 +3265,10 @@ en:
       share: "Share"
       text_empty_state_description: "The work package has not been shared with anyone yet."
       text_empty_state_header: "No shared users"
+      text_user_limit_reached:
+        zero: "This instance has reached its user limit, you can’t share with new users. Please contact your administrator to add more members."
+        one: "Only 1 member left to reach the user limit of the instance. Please contact your administrator."
+        other: "Only %{count} members left to reach the user limit of the instance. Please contact your administrator."
```

Comment:
Unless you already talked about this, it'd be good to check with product about this copyright. Based on the PR description note, users can still invite (but invited users can't sign in). However, the `zero` case here would indicate to me I can't

Reply:
I doubled checked with product and we adjusted the texts. 👍